x-pack/plugins/ml/public/application/explorer/components/explorer_map_container/embedded_map.tsx

```diff
@@ -0,0 +1,145 @@
+  } = useMlKibana();
+
+  if (!embeddablePlugin) {
+    throw new Error('Embeddable start plugin not found');
+  }
+  if (!mapsPlugin) {
+    throw new Error('Maps start plugin not found');
+  }
+
+  const factory: any = embeddablePlugin.getEmbeddableFactory(MAP_SAVED_OBJECT_TYPE);
+
+  const input: MapEmbeddableInput = {
+    id: uuid.v4(),
+    attributes: { title: '' },
+    filters: [],
+    hidePanelTitles: true,
+    refreshConfig: {
+      value: 0,
+      pause: false,
+    },
+    viewMode: ViewMode.VIEW,
+    isLayerTOCOpen: false,
+    hideFilterActions: true,
+    // Zoom Lat/Lon values are set to make sure map is in center in the panel
+    // It will also omit Greenland/Antarctica etc. NOTE: Can be removed when initialLocation is set
+    mapCenter: {
+      lon: 11,
+      lat: 20,
+      zoom: 1,
+    },
+    // can use mapSettings to center map on anomalies
+    mapSettings: {
+      disableInteractive: false,
+      hideToolbarOverlay: true,
+      hideLayerControl: false,
+      hideViewControl: false,
+      // Doesn't currently work with GEO_JSON. Will uncomment when https://github.com/elastic/kibana/pull/88294 is in
+      // initialLocation: INITIAL_LOCATION.AUTO_FIT_TO_BOUNDS, // this will startup based on data-extent
+      // autoFitToDataBounds: true, // this will auto-fit when there are changes to the filter and/or query
+    },
+  };
+
+  // Update the layer list  with updated geo points upon refresh
+  useEffect(() => {
+    if (embeddable && !isErrorEmbeddable(embeddable) && anomalies && anomalies.length > 0) {
+      layerList.current = [
+        layerList.current[0],
+        getMLAnomaliesActualLayer(anomalies),
+        getMLAnomaliesTypicalLayer(anomalies),
+      ];
+      embeddable.setLayerList(layerList.current);
+    }
+  }, [embeddable, anomalies]);
+
+  useEffect(() => {
+    async function setupEmbeddable() {
+      if (!factory) {
+        throw new Error('Map embeddable not found.');
+      }
+      const embeddableObject: any = await factory.create({
+        ...input,
+        title: 'Explorer map',
+      });
+
+      if (embeddableObject && !isErrorEmbeddable(embeddableObject)) {
+        const basemapLayerDescriptor = mapsPlugin
+          ? await mapsPlugin.createLayerDescriptors.createBasemapLayerDescriptor()
+          : null;
+
+        if (basemapLayerDescriptor) {
+          layerList.current = [basemapLayerDescriptor];
+          await embeddableObject.setLayerList(layerList.current);
+        }
+      }
+
+      setEmbeddable(embeddableObject);
+    }
+
+    setupEmbeddable();
+    // we want this effect to execute exactly once after the component mounts
+  }, []);
+
+  // We can only render after embeddable has already initialized
+  useEffect(() => {
+    if (embeddableRoot?.current && embeddable) {
+      embeddable.render(embeddableRoot.current);
+    }
+  }, [embeddable, embeddableRoot, anomalies]);
+
+  return (
+    <div
+      className={'mlFieldDataCard__geoContent'}
```

Comment:
I think we can remove this className here

Reply:
Removed 👍